oak_functions/client/rust/src/rekor.rs

```diff
@@ -159,10 +206,85 @@ pub fn verify_rekor_signature(
     pem_encoded_public_key_bytes: &[u8],
 ) -> anyhow::Result<()> {
     let signature_bundle = rekor_signature_bundle(log_entry_bytes)?;
+
+    verify_signature(
+        &signature_bundle.base64_signature,
+        &signature_bundle.canonicalized,
+        pem_encoded_public_key_bytes,
+    )
+    .context("failed to verify signedEntryTimestamp of the Rekor LogEntry")
```

Comment:
nit: failed -> Failed and add . at the end. Also for error messages below, but I am not 100% sure what our convention is.

Reply:
FWIW I think we should use consistent format for all errors, regardless of whether they are final or not. But we can discuss that separately.